internal/pkg/term/progress/cloudformation.go

```diff
@@ -17,11 +17,16 @@ import (
 	"golang.org/x/sync/errgroup"
 )
 
-// StackSubscriber is the interface to subscribe channels to a CloudFormation stack stream event.
+// StackSubscriber is the interface to subscribe to a CloudFormation stack event stream.
 type StackSubscriber interface {
 	Subscribe() <-chan stream.StackEvent
 }
 
+// StackSetSubscriber is the interface to subscribe channels to a CloudFormation stack set operation event stream.
```

Comment:
nit for consistency (alternatively, the other comment can be edited instead):
```suggestion
// StackSetSubscriber is the interface to subscribe channels to a CloudFormation stack set event stream.
```

Reply:
Done!